src/framework/ui/input/input.component.tsx

```diff
@@ -382,12 +382,17 @@ const styles = StyleSheet.create({
     flexGrow: 1,
     flexShrink: 1,
     flexBasis: 'auto',
+    textAlign: 'left',
```

Comment:
This must be auto, input works like this:
if the first character has RTL character in it then it change the text align to right
if not it then change the text align to left
by default it varies by the placeholder...